src/errors/specification-file.ts

```diff
@@ -34,15 +34,15 @@ export class ErrorLoadingSpec extends Error {
     super();
     if (from === 'file') {
       this.name = 'error loading AsyncAPI document from file';
-      this.message = `${param} is an invalid file path`;
+      this.message = `${param} file does not exist.`;
     }
     if (from === 'url') {
-      this.name = 'error loading AsyncAPI docuement from url';
-      this.message = `${param} is an invalid url`;
+      this.name = 'error loading AsyncAPI document from url';
+      this.message = `${param} url does not exist.`;
```

Comment:
We can't conclude that the URL does not exist but that we can't access it. Maybe something like this:

```suggestion
      this.message = `Failed to download ${param}.`;
```